include/seqan3/core/detail/strong_type.hpp

```diff
@@ -492,4 +493,30 @@ class strong_type
     value_t value;
 };
 
+/*!\name Formatted output
+ * \relates seqan3::detail::strong_type
+ * \{
+ */
+
+/*!\brief Formatted output to a seqan3::detail::debug_stream_type.
+ * \tparam char_t The char type of the seqan3::detail::debug_stream_type.
+ * \tparam strong_type_t The strong type to print; must model seqan3::detail::strong_type_specialisation.
+ *
+ * \param[in,out] stream The output stream.
+ * \param[in] value The strong typed value to print.
+ *
+ * \details
+ *
+ * Prints the stored value of the given strong type.
+ *
+ * \returns `stream_t &` A reference to the given stream.
+ */
+template <typename char_t, strong_type_specialisation strong_type_t>
+debug_stream_type<char_t> & operator<<(debug_stream_type<char_t> & stream, strong_type_t && value)
```

Comment:
Shouldn't this also require that the underlying type is streamable?
Edit: Damn but we still have this bug that "everything is streamable" right?